### PR TITLE
Add automation doc

### PR DESCRIPTION
**Pull request by robobario**
Add automation doc so non-admin users know what secrets are available for automation across repositories

### CODE REVIEW

File: AUTOMATION.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- Read access to metadata for all current and future repositories owned by this organization
+- Read and Write access to code and pull requests for all current and future repositories owned by your organization
+
+# Actions secrets and variables
+
+The Organization sets some Actions secrets so they are available to all repositories
+
+| Name                 | Description                                                         | Example       |
+|----------------------|---------------------------------------------------------------------|---------------|
+| DEPENDENCY_TOKEN     | ???                                                                 | secret :lock: |
```
Comment (robobario) [inline AUTOMATION.md]: this looks to be used by console to merge dependabot PRs, is it pointing at the Release PAT?

Comment (MikeEdgar) [reply AUTOMATION.md]: No, this is a separate token with less access than the release PAT.

File: AUTOMATION.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Read access to actions variables, metadata, and secrets for all current and future repositories owned by this
+  organization
+- Read and Write access to code, issues, and pull requests for all current and future repositories owned by this
+  organization
+
+### Sonar PAT
+
+The sonar PAT requires permissions:
+- Read access to metadata for all current and future repositories owned by this organization
+- Read and Write access to code and pull requests for all current and future repositories owned by your organization
```
Comment (MikeEdgar) [inline AUTOMATION.md]: The Sonar token is defined/configured in Sonar Cloud itself, rather than being a GitHub PAT.

File: AUTOMATION.md (hunk 1 above)
Comment (MikeEdgar) [reply AUTOMATION.md]: It has read access to metadata, and read/write to code + pull requests.

**Review by MikeEdgar — COMMENTED** [review]
Added a few comments.

File: AUTOMATION.md (hunk 1 above)
Comment (robobario) [reply AUTOMATION.md]: thanks I was mixed up I had it in my mind the PAT had sonar in it's name but it's called `streamshub-dependency-token`, d'oh!

Comment (robobario) [reply AUTOMATION.md]: updated the doc

File: AUTOMATION.md (hunk 2 above)
Comment (robobario) [reply AUTOMATION.md]: Updated. Does it have an expiry or anything else worth noting?

Comment (MikeEdgar) [reply AUTOMATION.md]: No expiry and nothing really special about it. As far as I can tell tokens in Sonar are all the same, just giving access to act on behalf of their owner.

**Review by MikeEdgar — APPROVED** [review]
Looks good. Also note that the release and dependency tokens will expire at the end of this year. I typically am notified ahead of time and renew them.

**Comment by robobario** [comment]
shuffled to .github-private instead :+1:  it's now visible on https://github.com/streamshub for members. Still imagine this repo will be useful, @tomncooper was talking about getting the purpose of streamshub nailed down and documented. Creating a `profile/README.md` here will make it publicly visible on the github org page. It should be a good place to have it upstream and visible to users.

**Pull request closed without merging**